### PR TITLE
Молоткова С.С. 381806-1. ЛР3.

### PR DESCRIPTION
**Pull request by SMolotkova**
### Pull Request Readiness Checklist

See details at [code review]( https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/lab-guide/lab-1-setup.md#%D1%80%D0%B5%D1%86%D0%B5%D0%BD%D0%B7%D0%B8%D1%8F-%D0%BA%D0%BE%D0%B4%D0%B0)

- [ ] Code is well documented and more information can be seen on the [website](https://devtools-course-practice-2021.readthedocs.io/en/latest/modules.html) (recommended)
- [x] Make sure that the testing has passed successfully on [GitHub Actions](https://github.com/UNN-ITMM-Software/devtools-course-practice/actions)
- [x] Get two approve from your fellow students
- [x] Take a review from one of the administrators

### CODE REVIEW

**Review by BelikJulia — APPROVED** [review]
Все отлично

File: modules/params-triangle/CMakeLists.txt [hunk 1]
```diff
@@ -4,10 +4,18 @@ get_filename_component(DIR_NAME ${CMAKE_CURRENT_LIST_DIR} NAME)
...
 # Add all submodules
 add_subdirectory(src)
 add_subdirectory(test)
+add_subdirectory(application)
+
+#############################################
+##### Testing
+#############################################
+
+include("CTestTests.txt")
```
Comment (streltsova-yana) [inline modules/params-triangle/CMakeLists.txt:21]: Необходимо добавить пустую строку в конец файла

File: modules/params-triangle/application/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+add_executable(${target} ${srcs})
+set_target_properties(${target} PROPERTIES
+    OUTPUT_NAME ${MODULE}
+    LABELS "${MODULE};Application")
+
+target_link_libraries(${target} ${LIBRARY})
+if (UNIX)
+  target_link_libraries(${target} ${CMAKE_THREAD_LIBS_INIT})
+endif (UNIX)
```
Comment (streltsova-yana) [inline modules/params-triangle/application/CMakeLists.txt:15]: Необходимо добавить пустую строку в конец файла

**Review by streltsova-yana — CHANGES_REQUESTED** [review]

File: modules/params-triangle/CMakeLists.txt (hunk 1 above)
Comment (SMolotkova) [reply modules/params-triangle/CMakeLists.txt]: Добавила

File: modules/params-triangle/application/CMakeLists.txt (hunk 2 above)
Comment (SMolotkova) [reply modules/params-triangle/application/CMakeLists.txt]: Добавила

**Review by streltsova-yana — APPROVED** [review]

**Pull request merged**